CHANGELOG.next.toml

```diff
@@ -23,3 +23,19 @@ message = "Implement `Ord` and `PartialOrd` for `DateTime`."
 author = "henriiik"
 references = ["smithy-rs#2653"]
 meta = { "breaking" = false, "tada" = false, "bug" = false }
+
+[[smithy-rs]]
+message = """
+<details>
+<summary>Breaking change in how event stream signing works (click to expand more details)</summary>
+
+This change will only impact you if you are wiring up their own event stream signing/authentication scheme. If you're using `aws-sig-auth` to use AWS SigV4 event stream signing, then this change will **not** impact you.
```

Comment:
We should say in the summary how people can know if this affects them e.g. "This impacts you if <reasoning>".

Reply:
It seems like a lot to include in the summary line.